### ISSUE TITLE
Missing title in get_current_track_info

### ISSUE THREAD

**Issue by LionelCons** [SoCo/SoCo#903]
When investigating a missing title in my audio software, I traced it back to some incorrect output from `get_current_track_info`.

In short, the following code:
```
print(device.get_current_track_info())
```
gives:
```
{'title': '', 'artist': 'Angel City', 'album': 'Face to Face', 'album_art': '...', 'position': '0:01:59', 'playlist_position': '9', 'duration': '0:03:14', 'uri': '...', 'metadata': '<DIDL-Lite xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:upnp="urn:schemas-upnp-org:metadata-1-0/upnp/" xmlns:r="urn:schemas-rinconnetworks-com:metadata-1-0/" xmlns="urn:schemas-upnp-org:metadata-1-0/DIDL-Lite/"><item id="-1" parentID="-1" restricted="true"><res protocolInfo="x-file-cifs:*:audio/mpeg:*" duration="0:03:14">...</res><r:streamContent></r:streamContent><upnp:albumArtURI>...</upnp:albumArtURI><dc:title>Waiting for the World</dc:title><upnp:class>object.item.audioItem.musicTrack</upnp:class><dc:creator>Angel City</dc:creator><upnp:album>Face to Face</upnp:album><upnp:originalTrackNumber>9</upnp:originalTrackNumber><r:albumArtist>Angel City</r:albumArtist></item></DIDL-Lite>'}
```

The track title is present in the metadata but not in the dict.

I'm using the latest version from `pip`: 0.26.3.

**Comment by jjlawren** [comment]
I'm investigating this in a downstream project now. It's likely a side-effect of #896 in regards to local music via CIFS/Samba file shares. You redacted the value, but the `uri` is very important here.

**Comment by LionelCons** [comment]
Here is the `uri` in the dict, less redacted:
```
'uri': 'x-file-cifs://xxx/music/Angel City - Face to Face/09 - Angel City - Waiting for the World.mp3'
```

**Comment by jjlawren** [comment]
Yep, that's the same issue I'm seeing elsewhere. Working on a fix.

**Comment by jjlawren** [comment]
@LionelCons would you be willing to test #906 and provide feedback?

**Comment by LionelCons** [comment]
I've tried with the new `core.py` and the title information is now back.

Thanks!

**Comment by pwt** [comment]
Fix released in SoCo v0.26.4. Thanks for spotting this issue.
